number_of_objects: 1
spawn_center:
  pos_x: 1.3
  pos_y: 0
  pos_z: 0.1
spawn_radius:
  x_length: 0.25
  y_length: 0.25
object_types:
  cube: True
  cylinders: True
  spheres: True